understanding/22/target-size-minimum.html

```diff
@@ -100,14 +101,14 @@ <h2>Intent of Target Size (Minimum)</h2>
     </section>
     <section id="benefits">
       <h2>Benefits of Target Size (Minimum)</h2>
-      <p>Having targets with sufficient size, or failing that, sufficient target spacing can help all users who may have difficulty in confidently targeting or operating small controls. Users who benefit include, but are not limited to:</p>
+      <p>Having targets with sufficient size - or failing that, sufficient target spacing - can help all users who may have difficulty in confidently targeting or operating small controls. Users who benefit include, but are not limited to:</p>
       <ul>
-        <li>Users who use a mobile device where the touch screen is the primary mode of interaction;</li>
-        <li>Users with mobility impairments such as hand tremors;</li>
-       <li>Users who use a device in environments where they are exposed to shaking such as public transportation;</li>
-       <li>Users who have difficulty with fine motor movements;</li>
-       <li>Users who access a device using one hand;</li>
-       <li>Users with large fingers, or who are operating the device with only a part of their finger or knuckle.</li>
+        <li>People who use a mobile device where the touch screen is the primary mode of interaction;</li>
+        <liPeople using mouse, stylus or touch input who have mobility impairments such as hand tremors;</li>
```

Comment:
missing closing `>` for the opening list item tag

Reply:
good catch, thanks! Corrected.